src/EmailUsers.py

```diff
@@ -0,0 +1,35 @@
+import yagmail
+from decouple import config
+
+
+class EmailUsers:
+    def __init__(self):
+        sender = config('USER')
+        key = config('KEY')
+        self.yag = yagmail.register(sender, key)
+        self.yag = yagmail.SMTP()
+        # another workaround is to call yagmail.register('gmail', 'password') once and have it registered
+        # via the keyring library
+
+    # sends email to given user 'to_send'
+    def send_email(self,
+                   body,
+                   subject,
+                   to_send):
+        try:
+            self.yag.send(to_send, subject, body)
+        except yagmail.error.YagAddressError:
+            print("Address given is in an invalid format")
+        except yagmail.error.YagInvalidEmailAddress:
+            print("Email address does not exist")
```

Comment:
Same here please

Reply:
these aren't comments, should I still take every print statement after the except clauses?

Reply:
oh sorry, I meant the same as the comment where I mention to raise exceptions instead of printing